src/main/java/dk/xam/jbang/cli/JbangAlias.java

```diff
@@ -0,0 +1,54 @@
+package dk.xam.jbang.cli;
+
+import dk.xam.jbang.Settings;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "alias", description = "Manage aliases.", subcommands = { JbangAliasAdd.class,
+		JbangAliasList.class,
```

Comment:
instead of subclasses per subcommand i think it would be simpler to use methods as subcommands - see https://picocli.info/#_subcommands_as_methods

that would make this class simpler/more readable.

Reply:
I can try that, but I think the gains will be very small, a couple of lines at most.

Reply:
Done

Reply:
i find that much nicer - 3 less classes :)